src/content/docs/adventure/contributing.mdx

```diff
@@ -0,0 +1,7 @@
+---
+title: Contributing to the Documentation
+description: How to contribute to the Documentation.
+---
+
+This site is only preserved to keep existing links in-tact. In order to contribute to the PaperMC documentation, read
+the [CONTRIBUTING.md](https://github.com/PaperMC/docs/blob/main/CONTRIBUTING.md) in the documentation's GitHub repository.
```

Comment:
Happy to just yeet this page, idc if links are broken.

Reply:
could be redirected via the `_redirects` file